### PR TITLE
OSDOCS#10082 Reworking AWS topic map/TOC

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-10082

Link to docs preview:

[Installing a cluster](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-default.html)
[Installing a cluster into a China region](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-china.html)
[Installing a cluster with customizations](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-customizations.html)
[Installing a cluster into a government region](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-government-region.html)
[Installing a cluster with compute nodes on Local Zones](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-localzone.html)
[Installing a cluster with network customizations](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-network-customizations.html)
[Installing a private cluster](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-private.html)
[Installing a cluster into a secret region](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-secret-region.html)
[Installing a cluster into an existing VPC](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-vpc.html)
[Installing a cluster with compute nodes on Wavelength Zones](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-wavelength-zone.html)
[Installing a cluster on a restricted network](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-restricted-networks-aws-installer-provisioned.html)

[Installing a cluster (UPI)](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/upi/installing-aws-user-infra.html)
[Installing a cluster on a restricted network (UPI)](https://74484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/upi/installing-restricted-networks-aws.html)

QE not required as this is a structural change 
Received LGTM from Stephanie Stout (content strategist) via Slack
PM not required as this structure exactly mirrors the previous changes to vSphere

### CODE REVIEW

File: installing/installing_aws/ipi/installing-aws-network-customizations.adoc [hunk 1]
```diff
@@ -108,7 +108,7 @@ include::modules/nw-modifying-operator-install-config.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-For more information on using a Network Load Balancer (NLB) on AWS, see xref:../../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc#configuring-ingress-cluster-traffic-aws-network-load-balancer[Configuring Ingress cluster traffic on AWS using a Network Load Balancer].
+For more information on using a Network Load Balancer (NLB) on AWS, see xref:../../../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc#configuring-ingress-cluster-traffic-aws-network-load-balancer[Configuring Ingress cluster traffic on AWS using a Network Load Balancer].
```
Comment (jeana-redhat) [inline installing/installing_aws/ipi/installing-aws-network-customizations.adoc:111]: Assuming the link title is correct, it goes here:
```suggestion
For more information on using a Network Load Balancer (NLB) on AWS, see xref:../../../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc#nw-configuring-ingress-cluster-traffic-aws-network-load-balancer_configuring-ingress-cluster-traffic-aws[Configuring Ingress cluster traffic on AWS using a Network Load Balancer].
```

**Review by jeana-redhat — COMMENTED** [review]
Phew good job! I mentioned a few places that add'l resources bullets could be trimmed to match guidelines (only link text). I notice that there is not very good adherence to this in the install books overall, so probably this is a "later" task, unless you want to hit the handful I mentioned here.

Found a couple links that I think are off, but really not much considering the level of effort here.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: installing/installing_aws/modules
[the remark this answers is not in the document]
Comment (bscott-rh) [reply installing/installing_aws/modules:1]: Based on my testing, `ls` and `cd` worked for both forms of symbolic link.

File: _unused_topics/manually-creating-iam.adoc
[the remark this answers is not in the document]
Comment (bscott-rh) [reply _unused_topics/manually-creating-iam.adoc:1]: Great, I'll delete it

**Comment by bscott-rh** [comment]
Received LGTM from Stephanie Stout via Slack.

**Pull request merged**